backend/src/main/scala/cromwell/backend/BackendCacheHitCopyingActor.scala

```diff
@@ -10,8 +10,11 @@ object BackendCacheHitCopyingActor {
   final case class CopyingOutputsFailedResponse(jobKey: JobKey, cacheCopyAttempt: Int, failure: CacheCopyError)
 
   sealed trait CacheCopyError
-  final case class LoggableCacheCopyError(failure: Throwable) extends CacheCopyError
-  final case class MetricableCacheCopyError(failureCategory: MetricableCacheCopyErrorCategory) extends CacheCopyError
+  /** A cache hit copy was attempted but failed.  */
+  final case class CopyAttemptError(failure: Throwable) extends CacheCopyError
+  /** Copying was requested for a blacklisted cache hit, however the cache hit copying actor found the hit had already
+    * been blacklisted so no novel copy attempt was made. */
+  final case class BlacklistedError(failureCategory: MetricableCacheCopyErrorCategory) extends CacheCopyError
```

Comment:
```suggestion
  final case class BlacklistSkip(failureCategory: MetricableCacheCopyErrorCategory) extends CacheCopyError
```
[TOL] Is it really an error if we use the blacklist to avoid having an error?

Reply:
Agree, though now the name of the base trait seems off...